docs/docs/glossary/infrastructure-as-code.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Infrastructure as Code
+disableTableOfContents: true
+---
+
+Learn what <q>Infrastructure as Code</q> means, and how you can use can use code to standardize and automate your Gatsby site.
+
+## What is <q>Infrastructure As Code?</q>
+
+_Infrastructure As Code_ is the practice of managing your development, testing, and production environments using configuration files or scripts. Provisioning and configuring environments individually can introduce errors or inconsistencies. You may, for example, find yourself running different versions of Node on your laptop and your production servers. Infrastructure As Code mimimizes this kind of drift and lets you automate the process of provisioning environments.
+
+Your configuration file describes what resources your project requires. If you're building an API, for example, you might create a configuration file that says, "Please install Node 12.16.2, npm 6.14.4, Express 4.17.1, and PostgreSQL 12.2 for Ubuntu Linux." For a Gatsby project, your configuration file may add plugins and themes.
+
+Configuration files, like other code files, are text. That means you can use version control software to store them and track changes to the environment. In short, infrastructure as code:
+
+- Creates consistent environments.
+- Saves time that would otherwise be spent setting up environments.
+- Reduces the risk of errors caused by mismatched environments.
+- Makes it easy to automate tasks.
+
+## Automating Gatsby site development with Gatsby Recipes
+
+[Gatsby Recipes](https://www.gatsbyjs.org/blog/2020-04-15-announcing-gatsby-recipes/) applies the infrastructure as code concept to front-end development. With Gatsby Recipes, you can automate common site building tasks, such as adding a plugin or test suite.
+
+To get started with Gatsby Recipes, upgrade to the latest version of Gatsby and the Gatsby CLI.
+
+```shell
+npm install -g gatsby-cli@latest
+npm install gatsby@latest
+```
+
+You can see the list of available recipes by using `gatsby recipes`. Use your up or down arrow keys to select a recipe, and press the _Enter_ or _Return_ key to run it.
+
+![A screenshot that contains a list of Gatsby recipes.](https://www.gatsbyjs.org/static/abf1101318022126e896af6222fb1cb7/b04e4/recipes-screenshot.png)
```

Comment:
I would make `infrastructure-as-code` a folder and copy this image into it. _This_ file would then be `index.md` in the same folder.